prow/plank/controller.go

```diff
@@ -511,6 +513,26 @@ func (c *Controller) syncPendingJob(pj prowapi.ProwJob, pm map[string]corev1.Pod
 	return c.prowJobClient.Patch(c.ctx, pj.DeepCopy(), ctrlruntimeclient.MergeFrom(&prevPJ))
 }
 
+func (c *Controller) syncAbortedJob(pj prowapi.ProwJob, pm map[string]corev1.Pod, _ chan<- prowapi.ProwJob) error {
+	if pj.Status.State != prowapi.AbortedState || pj.Complete() {
+		return nil
+	}
+
+	if pod, podExists := pm[pj.Name]; podExists {
+		client, ok := c.buildClients[pj.ClusterAlias()]
+		if !ok {
+			return fmt.Errorf("unknown cluster alias %q", pj.ClusterAlias())
+		}
+		if err := client.Delete(c.ctx, pod.DeepCopy()); err != nil && !kerrors.IsNotFound(err) {
```

Comment:
We need to respect the value of `AllowCancellations` in both plank and jenkins-operator. I don't think this is acceptable if `AllowCancellations` is false. This could share code with the `terminateDupes` function.

Reply:
`AllowCancellations` is deprecated and has a ` // TODO(fejta): delete this Mar 2020` comment on it.

Should I create a follow-up to remove it?

Reply:
😆 Yes that sounds good too.